api/src/main/java/io/opencensus/stats/NoopStats.java

```diff
@@ -116,21 +118,28 @@ public void setState(StatsCollectionState state) {
 
     @Override
     public MeasureMap newMeasureMap() {
-      return getNoopMeasureMap();
+      return newNoopMeasureMap();
     }
   }
 
   @Immutable
   private static final class NoopMeasureMap extends MeasureMap {
-    static final MeasureMap INSTANCE = new NoopMeasureMap();
+    private static final Logger logger = Logger.getLogger(NoopMeasureMap.class.getName());
+    private volatile boolean hasUnsupportedValues;
```

Comment:
don't need to make this volatile because the builder is not marked as ThreadSafe. Also this class is no longer Immutable.

Reply:
Done